disco_aws_automation/disco_rds.py

```diff
@@ -222,7 +222,7 @@ def modify_db_instance(self, instance_params, apply_immediately=True):
         logger.info("Updating RDS cluster %s", instance_params["DBInstanceIdentifier"])
         params = RDS.delete_keys(instance_params, [
             "Engine", "LicenseModel", "DBSubnetGroupName", "PubliclyAccessible",
-            "MasterUsername", "Port", "CharacterSetName", "StorageEncrypted"])
+            "MasterUsername", "Port", "CharacterSetName", "StorageEncrypted", "Tags"])
```

Comment:
It will throw an error because we are converting the `instance_params` dict into parameter pairs to the `self.client.modify_db_instance(..)` method. `Tags` is not one of the expected parameters.